docs/platform_management_plan/project_management.rst

```diff
@@ -181,4 +181,216 @@ meetings, where such topics are discussed and decided, see `Steering committees`
+  leads of all software modules always assign their sagas to the *roadmap project*.
+  All sagas in the *roadmap project* are extended with additional information
+  as e.g. start date and due date, to keep the status of the project always transparent.
+  Additionally, the main *SCORE* repository defines project wide milestones & releases,
+  that are visible in the roadmap as well.
+
+.. image:: _assets/roadmap_example.png
+    :width: 600
+    :alt: Roadmap example
+    :align: center
+
+Releases and milestones
+^^^^^^^^^^^^^^^^^^^^^^^^
+GitHub allows to define various milestones & releases for every repository. The definition of the milestones and releases is proposed by the *Technical Leads* and is approved by *Project Leads*.
+
+In the main *SCORE* project we use milestones to mark important stages of the project and map sagas or in some cases also other *GitHub Issues* to them.
+
+*Releases* are used for structuring of the development activities. Exact scheme for the releases of the *SCORE* will be provided here later.
+
+You can find "up to date" overview of the release plan and milestones in the following section `Score Releases <https://eclipse-score.github.io/score/score_releases/index.html>`_.
+
+Planning process
+----------------
+Generally, every team is responsible for planning and managing of its backlog.
+For small improvements or clarifications, you can create *GitHub Issue* with a exhaustive
+description and map it to the topic using labels. For small improvements/bugs
+in the software modules you should create *GitHub Issues* directly in the repository
+of the submodule. The project leads and committers of the corresponding software module,
+circle or community will check the issue and in case they will accept it, they will
+take it over to one of their *GitHub Projects*. In case, the topic, that you raise in the issue has a big impact on the platform, you can be asked by the committers to raise a *Contribution Request*.
+
+*Contribution Request* is the central concept to introduce any important changes
+to the project. It is described in more details in *Contribution Request* (TODO: link).
+In general, everyone who wants to provide something new to the project, e.g. a new feature
+or a tool, should provide an exhaustive description, requirements and in some cases
+also initial draft of the architecture as part of the *Contribution Request*.
+The *Contribution Requests* are regularly reviewed in the *Technical lead circle*
+and then get either accepted or declined.
+
+After the *Contribution Request* was accepted, then the *Pull Request* with the
+*Contribution Request* gets merged. The corresponding *GitHub Issue* gets a reference to the
+newly defined saga and gets closed. The saga is at the beginning in the state *"Draft"*. Please be aware, that "status" of the tickets is modelled in *GitHub Project* as *GitHub Issues* do not provide the possibility to define additional states.
```

Comment:
better: "newly defined saga which plans the implementation of the feature request"